node/pkg/ethereum/watcher.go

```diff
@@ -522,9 +545,9 @@ func (e *Watcher) Run(ctx context.Context) error {
 								zap.Stringer("current_block", ev.Number),
 								zap.Stringer("current_blockhash", currentHash),
 								zap.String("eth_network", e.networkName))
-							delete(e.pending, key)
```

Comment:
Why don't we want to delete this? If we don't, we will generate the log message and peg the count multiple times until it gets dropped for being too old. Also, if this is a "SECURITY CRITICAL" error, should this be more than a logger.Info?